client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -945,7 +952,8 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         fetchAllCountyUsers,
         tableTitle, 
         timeRangeFilter,
-        isBadgeInVisible
+        isBadgeInVisible,
+        changeTimeRangeFilter
```

Comment:
maybe move it to line 948 after `changeSearchFilter` and also move `changeStatusFilter` to 949

Reply:
changed it